client/App/Components/Main.js

```diff
@@ -86,6 +87,12 @@ class SwiperView extends React.Component{
           params={this.state} 
           showsButtons={false} 
           userId={this.props.route.userId}/>
+        <Recorder navigator={this.props.navigator} 
```

Comment:
you need to insert a line on line 50/51 that looks like the others but with the new index. This is the same index that will be used in recorder view file